src/main/java/step2/domain/LottoGenerator.java

```diff
@@ -0,0 +1,63 @@
+package step2.domain;
+
+import step2.dto.LottoDTO;
+import step2.dto.LottoListDTO;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoGenerator {
+
+    private static final int UPPER_BOUND = 45;
+    private static final int COUNT = 6;
+
+    public LottoListDTO generateLottoList(int amount) {
+        List<LottoDTO> lottoList = new ArrayList<>();
+        for (int count = 0; count < amount; count++) {
+            lottoList.add(generateLotto());
+        }
+        return new LottoListDTO(lottoList);
+    }
+
+    public LottoDTO generateLotto() {
+        return issueLotto(LottoNumberGenerator.issueNumbers(UPPER_BOUND, COUNT));
+    }
+
+    public LottoDTO generateLotto(String values) {
+        List<Integer> numbers = Arrays.stream(values.split(","))
+                .map(String::trim)
+                .map(Integer::parseInt)
+                .sorted()
+                .collect(Collectors.toList());
+        return issueLotto(numbers);
+    }
+
+    private LottoDTO issueLotto(List<Integer> numbers) {
+        if (numbers.size() != COUNT) {
+            throw new IllegalArgumentException();
+        }
+        if (!areNumbersValid(numbers)) {
+            throw new IllegalArgumentException();
+        }
+        if (!areNumbersUnique(numbers)) {
+            throw new IllegalArgumentException();
+        }
+        return new LottoDTO(numbers);
+    }
+
+    private boolean areNumbersUnique(List<Integer> numbers) {
```

Comment:
```suggestion
    private boolean unique(List<Integer> numbers) {
```

엘레강트 오브젝트의 관점에 따라 boolean 을 리턴하는 메서드는 형용사형으로 해주시면 될 것 같습니다.